sql/src/test/java/org/opensearch/sql/sql/antlr/MatchBoolPrefixParserTest.java

```diff
@@ -0,0 +1,18 @@
+package org.opensearch.sql.sql.antlr;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.opensearch.sql.common.antlr.SyntaxParserTestBase;
```

Comment:
is this used?

Reply:
It used to be. 

If it's not, I'm surprised `checkStyleTest` didn't say anything about it.